yb-voyager/src/migassessment/sizing.go

```diff
@@ -451,46 +522,34 @@ Returns:
 	sizeInGB: size of source object in gigabytes
 */
 func bytesToGB(sizeInBytes float64) float64 {
-	return sizeInBytes / (1024 * 1024 * 1024)
+	sizeInGB := sizeInBytes / (1024 * 1024 * 1024)
+	// any value less than a 1 MB is considered as 0
+	if sizeInGB < 0.001 {
+		return 0
+	}
+	return sizeInGB
 }
 
-func createConnectionToExperimentData(targetYbVersion string) {
-	filePath := getExperimentFile(targetYbVersion)
+func createConnectionToExperimentData(assessmentMetadataDir string) {
+	filePath := getExperimentFile(assessmentMetadataDir)
 	err := ConnectExperimentDataDatabase(filePath)
 	if err != nil {
 		panic(err)
 	}
 }
 
-func getExperimentFile(targetYbVersion string) string {
-	filePath := "src/migassessment/resources/yb_" + strings.ReplaceAll(targetYbVersion, ".", "_") + "_source.db"
+func getExperimentFile(assessmentMetadataDir string) string {
 	if checkInternetAccess() {
-		remoteFileExists := checkFileExistsOnRemoteRepo(filePath)
-		if remoteFileExists {
-			filePath = strings.ReplaceAll(filePath, "src/migassessment/resources/", baseDownloadPath)
-		} else {
-			// check if local file exists
-			fmt.Println(filePath)
-			isFileExist := utils.FileOrFolderExists(filePath)
-
-			if !isFileExist {
-				panic("file doesn't exist")
-			}
-		}
+		checkAndDownloadFileExistsOnRemoteRepo(assessmentMetadataDir)
 	} else {
-		// no network access
-		fmt.Println("No network access. Checking file locally...")
-		// check if local file exists
-		isFileExist := utils.FileOrFolderExists(filePath)
-		if !isFileExist {
-			panic("file doesn't exist")
-		}
+		_ = os.WriteFile(assessmentMetadataDir+fileName, experimentData20240, 0644)
 	}
-	return filePath
+	return assessmentMetadataDir + fileName
 }
 
-func checkFileExistsOnRemoteRepo(fileName string) bool {
-	remotePath := "https://raw.githubusercontent.com/yb-voyager/sshaikh/mat/yb-voyager/" + fileName
+func checkAndDownloadFileExistsOnRemoteRepo(assessmentMetadataDir string) bool {
+	remotePath :=
```

Comment:
please add a comment above this telling the standard format for raw content API of github

Reply:
done